connectivity/netsocket/include/netsocket/TCPSocket.h

```diff
@@ -145,6 +145,14 @@ class TCPSocket : public InternetSocket {
     nsapi_size_or_error_t recvfrom(SocketAddress *address,
                                    void *data, nsapi_size_t size) override;
 
+    virtual nsapi_size_or_error_t sendmsg(const SocketAddress &address,
```

Comment:
Existing code (and recommended style) omits `virtual` with `override`.

Reply:
Done